### PR TITLE
Refactor the whole webapp structure

### PR DESCRIPTION
**Pull request by rafaellsma**
Review commit by commit could be easier.

### CODE REVIEW

File: vitaes-webapp/src/AddTemplate/index.js [hunk 1]
```diff
@@ -129,7 +130,7 @@ class AddTemplate extends Component {
           </a>
         </div>
         <div className="Base-button">
-          <a href="#" onClick={() => {}}>Add New Param</a>
+          <a href="#" onClick={() => { }}>Add New Param</a>
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/AddTemplate/index.js]: Is this standard syntax for empty lambdas?

**Comment by rafaellsma** [comment]
Look the new folders structure:
- each component is a module and children components are module inside of its parents. 
- Shared components must to be in shared folder related on its module tree.
- Component module and files must to be uppercase.
- Pure js module and files must to be lowercase.

File: vitaes-webapp/src/AddTemplate/index.js (hunk 1 above)
Comment (rafaellsma) [reply vitaes-webapp/src/AddTemplate/index.js]: For my prettier, yes.

**Comment by rafaellsma** [comment]
Actually the imports have 3 parts:
1. First import of librarys
2. Imports directly from root folder.
3. Imports from current folder('./')

> import React from 'react'
> import PropTypes from 'prop-types'
>
>import getHostname from 'utils/getHostname'
>
>import ChildComponent from './ChildComponent'
>import './component.css'

File: vitaes-webapp/src/Builder/CvItemForm/index.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+import { Alert, Button, Card } from 'react-bootstrap';
+
+import { strings } from 'i18n/strings';
+import capitalize from 'utils/capitalize';
+import validateDate from 'utils/validateDate';
+
+import { fieldsDef, updateFields } from './../shared/fields';
+import CvField from './CvField'
+
+updateFields()
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/CvItemForm/index.js]: semicolon here

File: vitaes-webapp/src/Builder/CvItemForm/index.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
+import React, { Component } from 'react';
+import { toast } from 'react-toastify';
+import { Alert, Button, Card } from 'react-bootstrap';
+
+import { strings } from 'i18n/strings';
+import capitalize from 'utils/capitalize';
+import validateDate from 'utils/validateDate';
+
+import { fieldsDef, updateFields } from './../shared/fields';
+import CvField from './CvField'
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/CvItemForm/index.js]: semicolon here

File: vitaes-webapp/src/Builder/CvItemForm/index.js [hunk 4]
```diff
@@ -0,0 +1,255 @@
+import React, { Component } from 'react';
+import { toast } from 'react-toastify';
+import { Alert, Button, Card } from 'react-bootstrap';
+
+import { strings } from 'i18n/strings';
+import capitalize from 'utils/capitalize';
+import validateDate from 'utils/validateDate';
+
+import { fieldsDef, updateFields } from './../shared/fields';
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/Builder/CvItemForm/index.js]: why not ../shared/...?

File: vitaes-webapp/src/TemplateHub/index.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { toast } from 'react-toastify';
+
+import TemplateHubModel from './TemplateHubModel'
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/TemplateHub/index.js]: semicolon

File: vitaes-webapp/src/Builder/CvItemForm/index.js (hunk 4 above)
Comment (rafaellsma) [reply vitaes-webapp/src/Builder/CvItemForm/index.js]: Can you explain it?

**Pull request merged**